datastore/tests/integration/query_test.py

```diff
@@ -0,0 +1,69 @@
+import requests
```

Comment:
I split `smoke_test.py`, hoping to make it easier to add other query-related tests